core/thread/inc/ROOT/RConcurrentHashColl.hxx

```diff
@@ -38,16 +42,23 @@ public:
       HashValue() = default;
       HashValue(const char *data, int len);
       ULong64_t const *Get() const { return fDigest; }
+
+      /// Return the hash value for this object
+      size_t Hash() const noexcept {
```

Comment:
It's *very* confusing that `HashValue` has a function `Hash`. To hash a hash value?! Please consider renaming `HashValue` to `Digest`.

Reply:
Technically it is a (shorter) hash value for a (longer) hash value and the outer name is pre-existing ...  but it's internal so we can probably change it.